File: Second_year/Java_Data_access/Exercise_to_practice/EjerciciosPropuestos_1.5.3/src/ejerciciospropuestos_1/pkg5/pkg3/Persona.java

package ejerciciospropuestos_1.pkg5.pkg3;

import java.io.Serializable;

/**
 *
 * @author Ignacio
 */
public class Persona implements Serializable{
    public Persona(){
        
    }
    public Persona(String nombre, String mail, int anyo){
        this.nombre = nombre;
        this.mail = mail;
        this.anyo = anyo;
    }
    private String nombre;
    private String mail;
    private int anyo;
    
    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    public String getNombre(){
        return nombre;
    }
    public void setMail(String mail){
        this.mail = mail;
    }
    public String getMail(){
        return mail;
    }
    public void setAnyo(int anyo){
        this.anyo = anyo;
    }
    public int getAnyo(){
        return anyo;
    }
}

File: First_year/C#/Exercises_to_practice/Ejercicios_propuestos_T9/Ejercicio_9_3_1b/Ejercicio_9_3_1b/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio_9_3_1b
{
    [Serializable]
    class Persona
    {
        string nombre;

        public void SetNombre(string nombre)
        {
            this.nombre = nombre;
        }
        public string GetNombre()
        {
            return nombre;
        }
    }
}
